src/Components/authenticate/SignUp.jsx

```diff
@@ -11,7 +10,7 @@ const SignUp = () => {
     const { name, value } = event.currentTarget;
     setDisplayName(value);
   };
-
+// TODO: Make a version of this setup with email and password
```

Comment:
don't think we actually need to do that, in fact I was thinking in a future version we could just get rid of signing in with credentials and just do anonymous sign-in (can send details about that later, but for now just going to leave it as a comment)

Reply:
Read above.